Containerfile.external-dns-operator-bundle

```
@@ -23,13 +23,21 @@ RUN ./update_bundle.sh
 # Based on Dockerfile.bundle file.
 FROM scratch
 
+# Define build arguments to receive dynamic values
+ARG SHORT_COMMIT
+ARG GET_URL
+
 # Core bundle labels.
 LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
 LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
 LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
 LABEL operators.operatorframework.io.bundle.package.v1=external-dns-operator
 LABEL operators.operatorframework.io.bundle.channels.v1=stable-v1.3,stable-v1
 LABEL operators.operatorframework.io.bundle.channel.default.v1=stable-v1
+LABEL io.openshift.build.commit.id=${SHORT_COMMIT}
+LABEL io.openshift.build.source-location=${GET_URL}
+LABEL io.openshift.build.commit.url=${GET_URL}/commit/${SHORT_COMMIT}
+
```

I'm curious. Will `COMMIT_SHA` param (which existed in `build-container` task before) work like this?

```suggestion
LABEL io.openshift.build.commit.sha=${COMMIT_SHA}
```